lib/calendar.dart

```diff
@@ -48,11 +48,11 @@ class ValueCalendarState {
     Timer timer2;
 
     var id = lid++;
-    var isFirst = true;
 
     check() {
       var current = getCurrent();
       if (current == null) {
+        func(defaultValue);
```

Comment:
not sure that I like this `func` stuff... First, the naming is a bit weird. Second, we don't know what it's supposed to do.